app/metrics.yaml

```diff
@@ -223,6 +223,74 @@ find_in_page:
     - telemetry-client-dev@mozilla.com
     expires: "2020-03-01"
 
+quick_action_sheet:
+  opened:
+    type: event
+    description: >
+      A user opened the quick action sheet UI
+    bugs:
+    - 1036
```

Comment:
The bug field here is the issue number in Github/Bugzilla for the request to add the telemetry

Reply:
Doh. Totally missed this field when copy-pasting 😓 sorry!